demo/demo_image_repo.py

```diff
@@ -263,12 +263,32 @@ def listen():
   # Register functions that can be called via XML-RPC, allowing users to add
   # target files to the image repository or to simulate attacks from a web
   # frontend.
-  server.register_function(add_target_to_imagerepo, 'add_target_to_supplier_repo')
+  server.register_function(add_target_to_imagerepo,
+      'add_target_to_supplier_repo')
```

Comment:
No good reason anymore, no. Will resolve in parallel with changes to web frontend.